themes/theme-ios11/pages/More/connector.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import { isUserLoggedIn } from '@shopgate/pwa-common/selectors/user';
+import logout from '@shopgate/pwa-common/actions/user/logout';
+import { QUICKLINKS_MENU } from '@shopgate/pwa-common/constants/MenuIDs';
+
+import { makeGetMenuById } from './selectors';
+
+const getMenuById = makeGetMenuById();
+const props = {
+  id: QUICKLINKS_MENU,
+};
+
+/**
+ * Maps the contents of the state to the component props.
+ * @param {Object} state The current application state.
+ * @return {Object} The extended component props.
+ */
+const mapStateToProps = state => ({
+  user: isUserLoggedIn(state) ? state.user.data : null,
+  isLoggedIn: isUserLoggedIn(state),
+  entries: {
+    quicklinks: getMenuById(state, props),
```

Comment:
This `mapStateToProps` introduces a tiny little performance issue, since it will always return a new object when the store updates. So we'd have some unnecessarily re-renderings.

Reply:
done